src/tools/compiletest/src/runtest.rs

```diff
@@ -4258,6 +4245,34 @@ impl<'test> TestCx<'test> {
                 V0_BACK_REF_RE.replace_all(&normalized, V0_BACK_REF_PLACEHOLDER).into_owned();
         }
 
+        // Normalize AllocId counter
```

Comment:
So this renumbers allocations, but they are still numbered? I guess I can see how but a comment explaining the motivation would still be good.